docs/application/native/guides/ui/dali/buttons.md

```diff
@@ -127,9 +127,9 @@ void HelloWorldExample::Create( Application& application )
 {
   CheckBoxButton button = CheckBoxButton::New();
   button.SetParentOrigin( ParentOrigin::CENTER );
-  button.SetLabelText( "Check" );
+  button.SetProperty( Button::Property::LABEL, "Check" );
   button.SetSize( 200, 40 );
-  button.SetBackgroundColor( Color::WHITE );
```

Comment:
Is SetBackgroundColor also deprecated? I can't find it from the list in ACR-1435

Reply:
Hi,
SetBackgroundColor was **not** deprecated and the functionality still exists in the codebase.
I updated the documentation so developers are guided to use the preferred method for setting the color.  They can still use the old API if they wish, it is not removed in the ACR.